### ISSUE TITLE
Create List of Strategies that are Missed In Risk Framework

### ISSUE THREAD

**Issue by rareweasel** [yearn/yearn-watch-legacy#248]
We realized some strategies were not included in the Risk Framework (RF). 

They were not add to the withdrawal queue. So when we filtered the strategies by name using a contract, it didn't find the strategy.

The ouput of this task is:

Create a list of strategies that are not found in the RF.

Maybe we can get the full list of strategies from the SDK (check how we get the data at the [home page](https://yearn.watch/)). 
Then when the RF filters the groups, create a new list with the strategies that were not included.

The new list might be in a new page, adding a link the RF page depending on the network.

Note:

There are 126 strategies In Fantom [home page](https://yearn.watch/network/fantom).
But there are only 31 in the [RF page](https://yearn.watch/network/fantom/risk).

Some missing strategies were retired from the vaults (queue index -1), but we need to know those strategies that are still active but are not in the RF.

**Comment by 0xkofee** [comment]
what would the values for these entries be for the strategies? @rareweasel 
```
"codeReviewScore": 2,
"testingScore": 2,
"auditScore": 5,
"protocolSafetyScore": 3,
"complexityScore": 4,
"teamKnowledgeScore": 2,
```
https://github.com/yearn/yearn-watch/blob/f906dabc15902357b78a9f2fe8b1ad0b7b678741/src/utils/config/risk-framework.json

**Comment by rareweasel** [comment]
> what would the values for these entries be for the strategies? @rareweasel
> 
> ```
> "codeReviewScore": 2,
> "testingScore": 2,
> "auditScore": 5,
> "protocolSafetyScore": 3,
> "complexityScore": 4,
> "teamKnowledgeScore": 2,
> ```
> 
> https://github.com/yearn/yearn-watch/blob/f906dabc15902357b78a9f2fe8b1ad0b7b678741/src/utils/config/risk-framework.json

That file contains all the scores for all the group of strategies in the Risk Framework (for now in ethereum and fantom).

**Comment by 0xkofee** [comment]
Hmm I did a second pass of the file and I think I misunderstood the ask. I thought we needed to add entries to [risk-framework.json](https://github.com/yearn/yearn-watch/blob/f906dabc15902357b78a9f2fe8b1ad0b7b678741/src/utils/config/risk-framework.json)

But now after looking more closely, each section has a `strategies` section which I guess one would need to add to for this ask. Is that correct? @rareweasel 

Or do we just simply make a list and post it here?

for example
```
{
            "id": "strategyleaguedaostaking",
            "network": "ethereum",
            "label": "League DAO Staking",
            "codeReviewScore": 2,
            "testingScore": 2,
            "auditScore": 5,
            "protocolSafetyScore": 3,
            "complexityScore": 1,
            "teamKnowledgeScore": 4,
            "criteria": {
                "nameLike": ["LeagueDAOStaking", "FiatDAOStakingYFI"],
                "strategies": [<ADD STRATEGY IDS HERE>],
                "exclude": []
            }
        },
```

**Comment by rareweasel** [comment]
> Hmm I did a second pass of the file and I think I misunderstood the ask. I thought we needed to add entries to [risk-framework.json](https://github.com/yearn/yearn-watch/blob/f906dabc15902357b78a9f2fe8b1ad0b7b678741/src/utils/config/risk-framework.json)
> 
> But now after looking more closely, each section has a `strategies` section which I guess one would need to add to for this ask. Is that correct? @rareweasel
> 
> Or do we just simply make a list and post it here?
> 
> for example
> 
> ```
> {
>             "id": "strategyleaguedaostaking",
>             "network": "ethereum",
>             "label": "League DAO Staking",
>             "codeReviewScore": 2,
>             "testingScore": 2,
>             "auditScore": 5,
>             "protocolSafetyScore": 3,
>             "complexityScore": 1,
>             "teamKnowledgeScore": 4,
>             "criteria": {
>                 "nameLike": ["LeagueDAOStaking", "FiatDAOStakingYFI"],
>                 "strategies": [<ADD STRATEGY IDS HERE>],
>                 "exclude": []
>             }
>         },
> ```

The output for this task is:

- Create a page in the webapp that list all the strategies that are missed in the Risk Framework (RF) based on what I described in the description.

It should be dynamic. It means I should be able to visit this new page to see if there are new missed strategies in the RF. 

If so, Storm or I will create a new issue (with the scores) to add it in the JSON file.

**Comment by magikadev** [comment]
Yeah, so would be better to create a new filter on home page for checking missed strategies.

When adding new ones to group, we make issues & PRs for it like before.

**Comment by 0xkofee** [comment]
alright I'll see if I can figure it out

**Comment by 0xkofee** [comment]
Ok, this was quite complex to figure out. I think I figured out how the current warnings filters works and before I start coding I wanted to run it by you both. Right now, vaults and strategies are "tagged" with various properties after we load them. These attributes are used by the frontend to know to put a red background around the vault and strategies(and to filter vaults).

These params are determined in `mapVaultApiDataToVault` and `mapVaultData`. Anyways, to follow the same pattern as the warnings filter I would need to add a call to the above mentioned functions to get all strategies the same way as the risk page and then compare it to the list of strategies on the frontpage. This would add many many more calls to the front page load. If this is acceptable I can do that but it doesn't seem very optimal. Btw, these would be blockchain method calls. So the slowness of the risk page would be added to the front page which is already super slow. Just wanted to get your thoughts...

@magic990619 @rareweasel

**Comment by rareweasel** [comment]
I would say, we should review the increment of time we would have implementing this approach.

The vaults page usually takes a time to load. So, idk how much time would be acceptable.

Is that approach the only one we are considering?

**Comment by 0xkofee** [comment]
I think following the pattern for how data is resolved for filters is a good idea in general and this is how it's done today. Another strategy would be to somehow load the vaults from the front page on the risk page so that we can find the delta... This way the front page is not having to load more data.

The long term approach seems to be to use subgraphs to populate everything we need so we don't need to spend time loading the vaults on the front page nor the vaults on the risk page.

maybe I'll just try implementing the solution and go from there...

**Comment by 0xkofee** [comment]
https://github.com/yearn/yearn-watch/pull/267/files

**Comment by saltyfacu** [comment]
Nice job!

With the new design, we will have a section called "Alerts". We can add these alerts to that part. 

A strategy that is not the risk framework is an important thing to tackle fast

**Comment by magikadev** [comment]
Solved in #288
